app/js/components/common/Header.js

```diff
@@ -112,7 +112,7 @@ export class Header extends Component {
 
                 <Link to="" activeClassName="active">
                     <li>
-                        <a href="#">Logout {' '}
+                        <a href="/openmrs/appui/header/logout.action?successUrl=openmrs">Logout {' '}
```

Comment:
The module lacked a logout link, hence the logout button was not working. To avoid raising a new ticket to add a single line link, we thought is best to include it in this PR and add it to the PR description.